subjects/blockchain/non-fungible-cats/audit/README.md

```diff
@@ -14,11 +14,11 @@
 
 ##### buy cat 1 at the listed price (10wei)
 
-###### The buy function functions properly
+###### The buy function works properly
 
 ##### transfer cat 1 to a secondary user
 
-###### The transfer function properly works
+###### The transfer function works properly
 
 ##### buy cat 2 at the listed price (10wei)
```

Comment:
It seems like a question, such as `Does the transfer work properly?`,  is missing after this action

Reply:
done